src/facts/wild-bergamot-bee-balm.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Wild Bergamot (bee balm)
```

Comment:
Can we just call this "Wild Bergamot" and say that its "often called bee balm" in the fact?